config/loom/index.ts

```diff
@@ -56,5 +60,43 @@ export function quiltPackage({
         ]);
       });
     }),
+    runTsd(),
   ]);
 }
+
+/**
+ * TSD allows us to run assertions against the types that are generated as part
+ * of the build
```

Comment:
Adding a bit of exposition about why this is a build step

```suggestion
 * TSD allows us to run assertions against the types that are generated as part
 * of the build. This is ran as a post-build step because tests the compiled
 * types in `packages/PACKAGENAME/build/ts` rather than source code.
```